Rlabkey/DESCRIPTION

```diff
@@ -1,18 +1,18 @@
-Package: Rlabkey
-Version: 2.2.5
-Date: 2019-03-04
-Title: Data Exchange Between R and 'LabKey' Server
-Author: Peter Hussey 
-Maintainer: Cory Nathe <cnathe@labkey.com>
-Description: The 'LabKey' client library for R makes it easy for R users to
-    load live data from a 'LabKey' Server, <http://www.labkey.com/>,
-    into the R environment for analysis, provided users have permissions
-    to read the data. It also enables R users to insert, update, and
-    delete records stored on a 'LabKey' Server, provided they have appropriate
-    permissions to do so.
-License: Apache License 2.0
-Copyright: Copyright (c) 2010-2018 LabKey Corporation
-LazyLoad: true
-Depends: httr, jsonlite
-LinkingTo: Rcpp
-Imports: Rcpp (>= 0.11.0)
+Package: Rlabkey
+Version: 2.2.5
+Date: 2019-03-15
+Title: Data Exchange Between R and 'LabKey' Server
+Author: Peter Hussey 
+Maintainer: Cory Nathe <cnathe@labkey.com>
+Description: The 'LabKey' client library for R makes it easy for R users to
+    load live data from a 'LabKey' Server, <http://www.labkey.com/>,
+    into the R environment for analysis, provided users have permissions
+    to read the data. It also enables R users to insert, update, and
+    delete records stored on a 'LabKey' Server, provided they have appropriate
+    permissions to do so.
+License: Apache License 2.0
+Copyright: Copyright (c) 2010-2018 LabKey Corporation
+LazyLoad: true
+Depends: httr, jsonlite
+LinkingTo: Rcpp
+Imports: Rcpp (>= 0.11.0)
```

Comment:
Please revert the changes to the `DESCRIPTION` file. It looks like you changed the line endings and Cory can update this when he's ready to push to CRAN.

Reply:
Hi Trey, 

Sure, but it would help to get a sense on the automated testing plan.  For example, when I fix those scenarios, my default would be to add test cases to that existing webDavTest.R file; however, I dont know that this is how tests should look.  As I noted above, since checking webDav operations should generally involving inspecting the results of the API call and checking files, the r-labkey-api.xml scheme may not be quite right either.  Any further thoughts there?

Reply:
I split up `r-labkey-api.xml` and there's now a `rlabkey-api-webdav.xml`. The test case that invokes it is currently disabled but if you comment out the `@Ignore` annotation from `RlabkeyTest.testRlabkeyWebDavApi` you'll be able to run the new webdav test cases: `./gradlew uiTest -Ptest=RlabkeyTest.testRlabkeyWebDavApi`
I think they cover all of the scenarios I mentioned and the failing test cases have the response I thought seemed to make sense.

Reply:
So I believe the basic problem is that R thinks the invalid webdav calls are succeeding.  for example:

response <- GET('https://prime-seq.ohsu.edu/_webdav/home/@files/fakeFile.txt')

reports a status of 200.  the body of the response has status_code = 404; however Rlabkey's processResponse doesnt pick up on that.  I'll investigate a little more.

Reply:
To illustrate more:
```

library(Rlabkey)
response <- GET('https://labkey.org/_webdav/fakeFile.txt')
#the response reports 200:
response$status_code 
#this is the actual JSON, which reports 404:
content(response)$status  

```
IF the header reported an error status, Rlabkey::processResponse would parse this and throw.  Because it doesnt, this looks like a successful request and that's not happening.

I am inclined to think either a) the server is doing the wrong thing (a more invasive change), or b) Rlabkey::processResponse() should be changed to cover this scenario, c) labkey.webdav should wrap processResponse() with a webdav-specific one that covers this scenario.

If anyone who knows more about HTTP requests has thoughts I would appreciate ideas.